datafusion/physical-expr/src/expressions/mod.rs

```diff
@@ -62,9 +62,12 @@ pub use crate::aggregate::sum_distinct::DistinctSum;
 pub use crate::aggregate::variance::{Variance, VariancePop};
 
 pub use crate::window::cume_dist::cume_dist;
+pub use crate::window::cume_dist::CumeDist;
+pub use crate::window::lead_lag::WindowShift;
 pub use crate::window::lead_lag::{lag, lead};
-pub use crate::window::nth_value::NthValue;
+pub use crate::window::nth_value::{NthValue, NthValueKind};
```

Comment:
Although the struct is public, but the mod is private. So i decide public less, only the needed struct.